lib/instrumentation/context.js

```diff
@@ -0,0 +1,97 @@
+'use strict'
+
+const crypto = require('crypto')
+
+function fromHex (hex) {
+  return Buffer.from(hex, 'hex')
+}
+
+function toHex (buffer) {
+  return buffer.toString('hex')
+}
+
+class TraceContext {
+  constructor ({ version, traceId, id, flags, parentId } = {}) {
+    // 1 byte
+    this.version = version
+    // 16 bytes
+    this.traceId = traceId
+    // 8 bytes
+    this.id = id
+    // 1 byte
+    this.flags = flags
+    // 8 bytes
+    this.parentId = parentId
```

Comment:
This doesn't seem to be used for anything at the moment?

Reply:
It's just a utility to verify structural integrity of context inheritance. Java does it this way too. We could do without it though. It'll be helpful if we eventually try to inherit spans from other spans though, rather than inheriting everything from the transaction context.